amanuensis/schema.py

```diff
@@ -11,6 +11,9 @@
     AttributeListValueSchema,
     InputTypeSchema,
     AssociatedUserSchema,
-    RequestStateSchema
+    RequestStateSchema,
+    SearchIsSharedSchema,
+    ProjectAssociatedUserSchema
 )
-
+#TODO fix this in userdataportalmodel
```

Comment:
what needs to be fixed here?

Reply:
theres a typo In userdataportalmodel.schema,
 .associated_user_roles_schema import AssociatedUserRoles
this is importing the model not schema